src/features/auth/signup/SetBirthday/DatePicker/DatePickerDropDown.web.test.tsx

```diff
@@ -0,0 +1,47 @@
+import mockdate from 'mockdate'
+import React from 'react'
+
+import {
+  MINIMUM_DATE,
+  CURRENT_DATE,
+  DEFAULT_SELECTED_DATE,
+} from 'features/auth/signup/SetBirthday/utils/fixtures'
+import { fireEvent, render } from 'tests/utils/web'
+
+import { DatePickerDropDown } from './DatePickerDropDown.web'
+
+const props = {
+  onChange: jest.fn(),
+  defaultSelectedDate: DEFAULT_SELECTED_DATE,
+  minimumDate: MINIMUM_DATE,
+}
+
+jest.mock('features/auth/settings')
+
+describe('<DatePickerDropDown />', () => {
+  beforeEach(() => {
+    mockdate.set(CURRENT_DATE)
+    props.onChange.mockReset()
+  })
+
+  it('should call onChange with undefined date when a date is not selected', () => {
+    const { getByTestId } = render(<DatePickerDropDown {...props} />)
+
+    fireEvent.change(getByTestId('select-Jour'), { target: { value: '' } })
+    fireEvent.change(getByTestId('select-Mois'), { target: { value: '' } })
+    fireEvent.change(getByTestId('select-Année'), { target: { value: '' } })
+
+    expect(props.onChange).toHaveBeenCalledWith(undefined) // first render trigger useEffect
```

Comment:
If the first expect is due to the useEffect, don't we want to check a second time then?

Reply:
Il n'y a pas de render au final car je ne change aucune value. C'était pour simuler l'état pas défaut.
Je me demande si c'est utile comme teste du coup ?

Reply:
Ah oui, je suis bête 😬 
En vrai, ça check explicitement la defaut value du date-picker, c'est bon pour moi !